### PR TITLE
update quickstart and llm_api_key_setup document

### PR DESCRIPTION
**Pull request by hangters**
### What problem does this PR solve?

update quickstart and llm_api_key_setup document

### Type of change

- [x] Documentation Update

### CODE REVIEW

File: docs/quickstart.mdx [hunk 1]
```diff
@@ -176,22 +176,25 @@ With the default settings, you only need to enter `http://IP_OF_YOUR_MACHINE` (*
...
+- [DeepSeek](https://platform.deepseek.com/api-docs/)
+- [Baichuan](https://www.baichuan-ai.com/home)
+- [VolcEngine](https://www.volcengine.com/docs/82379)
+- [Jina](https://jina.ai/reader/)
+- [OpenRouter](https://openrouter.ai/)
+- [StepFun](https://platform.stepfun.com/)
 
 :::note 
-RAGFlow also supports deploying LLMs locally using Ollama or Xinference, but this part is not covered in this quick start guide. 
+RAGFlow also supports deploying LLMs locally using Ollama or Xinference or LocalAI, but this part is not covered in this quick start guide.
```
Comment (writinwaters) [inline docs/quickstart.mdx]:
```suggestion
RAGFlow also supports deploying LLMs locally using Ollama, Xinference, or LocalAI, but this part is not covered in this quick start guide. 
```

File: docs/guides/llm_api_key_setup.md [hunk 2]
```diff
@@ -11,22 +11,25 @@ An API key is required for RAGFlow to interact with an online AI model. This gui
...
+- [DeepSeek](https://platform.deepseek.com/api-docs/)
+- [Baichuan](https://www.baichuan-ai.com/home)
+- [VolcEngine](https://www.volcengine.com/docs/82379)
+- [Jina](https://jina.ai/reader/)
+- [OpenRouter](https://openrouter.ai/)
+- [StepFun](https://platform.stepfun.com/)
 
 :::note
-If you find your online LLM is not on the list, don't feel disheartened. The list is expanding, and you can [file a feature request](https://github.com/infiniflow/ragflow/issues/new?assignees=&labels=feature+request&projects=&template=feature_request.yml&title=%5BFeature+Request%5D%3A+) with us! Alternatively, if you have customized or locally-deployed models, you can [bind them to RAGFlow using Ollama or Xinference](./deploy_local_llm.md).
+If you find your online LLM is not on the list, don't feel disheartened. The list is expanding, and you can [file a feature request](https://github.com/infiniflow/ragflow/issues/new?assignees=&labels=feature+request&projects=&template=feature_request.yml&title=%5BFeature+Request%5D%3A+) with us! Alternatively, if you have customized or locally-deployed models, you can [bind them to RAGFlow using Ollama or Xinference or LocalAI](./deploy_local_llm.md).
```
Comment (writinwaters) [inline docs/guides/llm_api_key_setup.md]:
```suggestion
If you find your online LLM is not on the list, don't feel disheartened. The list is expanding, and you can [file a feature request](https://github.com/infiniflow/ragflow/issues/new?assignees=&labels=feature+request&projects=&template=feature_request.yml&title=%5BFeature+Request%5D%3A+) with us! Alternatively, if you have customized or locally-deployed models, you can [bind them to RAGFlow using Ollama, Xinference, or LocalAI](./deploy_local_llm.md).
```

**Review by writinwaters — CHANGES_REQUESTED** [review]
Minor nit-picking changes.

File: docs/guides/llm_api_key_setup.md (hunk 2 above)
Comment (hangters) [reply docs/guides/llm_api_key_setup.md]: already changes

**Review by writinwaters — APPROVED** [review]
LGTM!

**Pull request merged**